cttportfolio.css

```
@@ -37,7 +41,7 @@ button{
 	text-transform:uppercase;
 	border:1px solid rgb(248,248,255);
 	font-weight:600;
-	border-radius:10px;
+	border-radius:5px;
```

```suggestion
	border-radius: 5px;
```
white spacing on all your values.